docs/core-study-details.md

```diff
@@ -27,6 +27,25 @@ for more information.
 
 # Table format
 
+Unless otherwise noted, column names correspond to a FHIR path relative to the resource
+table in question, with resources delineated by underscores. So, 'category_code' in the 
+Condition table would correspond to the code element inside of the category element of
+the Condition resource.
+
+Common exceptions to this rule:
+
+- `cnt` is a Cumulus specific notation for a column that contains counts of resources.
+- `age_at_visit` is calculated from a Patient's birthDate and the period.start.day of an Encounter
+- `postalcode_3` is calculated from a Patient's address.postalCode
+- The Encounter resource includes several elements from the referenced Patient (`gender`, `postalcode_3`,
+  and the US Core race and ethnicity extensions) that are commonly used in informatics analysis
+
+The core tables include all FHIR required/must support fields noted in the
+[FHIR resource profiles(http://hl7.org/fhir/us/core/STU4/artifacts.html#structures-resource-profiles).
+Additionally, there are fields that are useful to informatics analysis that are commonly available
+from EHRs, but are not guaranteed to be populated, so consult with your research partners if
+you are authoring a study using some of these data elements.
```

Comment:
@comorbidity drawing your attention to this text - not quite in db metadata, but would this work for external consumers, i.e. a DPH?

Reply:
@dogversioning Which part are you referring to? If you mean ZipCode with 3 digits, that's a HIPAA constraint. 
There has been an outstanding task to look at "census track data" but that has not yet been completed. ZipCode with 5 digits to my knowledge we do not have PHI clearance for, playing it safe here.

Reply:
@comorbidity this was more directed at answering the question that source metadata was trying to answer, i.e. 'i see this field name in the dashboard, where does it come from'?